tests/core/test_utils.py

```diff
@@ -0,0 +1,52 @@
+from pathlib import Path
+
+import numpy
+import pytest
+import trimesh
+
+from scenic.core.utils import loadMesh, repairMesh
+
+
+@pytest.mark.slow
+def test_mesh_repair():
+    plane_mesh = loadMesh(
+        path=Path(".").parent.parent.parent
```

Comment:
You could define an `assets` fixture in `tests/conftest.py` which returns a `Path` object for the assets folder. Then you could just use `assets / "meshes" / "classic_plane.obj.bz2"` for the path and it would work regardless of where the test is located.

Reply:
Good idea!